src/common/utils/currencyConvert.ts

```diff
@@ -20,4 +20,5 @@ export const getFiatValue = ({
   });
 };
 
-export const getSatValue = (balance: number) => `${balance} sats`;
+export const getSatValue = (balance: number) =>
```

Comment:
Should we switch this to `common` as well?

Reply:
Done!